src/EncapsulationKind.ts

```diff
@@ -12,27 +12,59 @@
 // {0x00, 0x14} -- DELIMIT_CDR, BIG_ENDIAN
 // {0x00, 0x15} -- DELIMIT_CDR, LITTLE_ENDIAN
 //
-// Verified with RTI Connext that 0x11 refers to decimal 11 (0x0B).
+// 7.6.2.1.2 RTPS encapsulation identifier
+// XCDR VERSION 1
+// {0x00, 0x00} -- CDR_BE, BIG_ENDIAN, FINAL
+// {0x00, 0x01} -- CDR_LE, LITTLE_ENDIAN, FINAL
+// {0x00, 0x00} -- CDR_BE, BIG_ENDIAN, APPENDABLE
+// {0x00, 0x01} -- CDR_LE, LITTLE_ENDIAN, APPENDABLE
+// {0x00, 0x02} -- PL_CDR_BE, BIG_ENDIAN, MUTABLE
+// {0x00, 0x03} -- PL_CDR_LE, LITTLE_ENDIAN, MUTABLE
+//
+// XCDR VERSION 2
+// {0x00, 0x06} -- CDR2_BE, BIG_ENDIAN, FINAL
+// {0x00, 0x07} -- CDR2_LE, LITTLE_ENDIAN, FINAL
+// {0x00, 0x08} -- D_CDR2_BE, BIG_ENDIAN, APPENDABLE
+// {0x00, 0x09} -- D_CDR2_LE, LITTLE_ENDIAN, APPENDABLE
+// {0x00, 0x0a} -- PL_CDR2_BE, BIG_ENDIAN, MUTABLE
+// {0x00, 0x0b} -- PL_CDR2_LE, LITTLE_ENDIAN, MUTABLE
 
 export enum EncapsulationKind {
+  // Both RTPS and ENC_HEADER enum values
   /** Plain CDR, big-endian */
-  CDR_BE = 0,
+  CDR_BE = 0x00,
   /** Plain CDR, little-endian */
-  CDR_LE = 1,
+  CDR_LE = 0x01,
   /** Parameter List CDR, big-endian */
-  PL_CDR_BE = 2,
+  PL_CDR_BE = 0x02,
   /** Parameter List CDR, little-endian */
-  PL_CDR_LE = 3,
+  PL_CDR_LE = 0x03,
+
+  // ENC_HEADER enum values
   /** Plain CDR2, big-endian */
-  CDR2_BE = 10,
+  CDR2_BE = 0x10,
```

Comment:
Make sure to do a major version bump on the next release

Reply:
Was this a set of typos? Or are we changing the enum values to make some other code easier?

Reply:
Yeah this was a mistake before. There was some confusion with the data we were checking this against. The hex values should be correct and align with the data we're seeing from customers